### PR TITLE
Fix `CVE-2024-47561` security vulnerability in the Avro library

### LINKED ISSUE #7243: Address `CVE-2024-47561` security vulnerability in the Avro library
**Description:**
There is a security vulnerability in the Apache Avro Java SDK which affects 1.11.3 and previous versions

Reference: https://nvd.nist.gov/vuln/detail/CVE-2024-47561

### PR DESCRIPTION
**Pull request by Nuvindu**

### CODE REVIEW

**Review by AzeemMuzammil — DISMISSED** [review]

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "avro"
-version = "1.0.0"
+version = "1.1.1"
```
Comment (Nuvindu) [inline ballerina/Ballerina.toml]:
```suggestion
version = "1.1.0"
```

File: gradle.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.lib
-version=1.0.1-SNAPSHOT
-ballerinaLangVersion=2201.9.0
+version=1.1.1-SNAPSHOT
+ballerinaLangVersion=2201.10.0
```
Comment (AzeemMuzammil) [inline gradle.properties]: do we need to bump the lang version?

Comment (Nuvindu) [reply gradle.properties]: Since this is required for lang 9.4 release, I will change the version back to 2201.9.0

Comment (Nuvindu) [inline gradle.properties]:
```suggestion
ballerinaLangVersion=2201.9.0
```

File: ballerina/Ballerina.toml [hunk 3]
```diff
@@ -18,29 +18,29 @@ graalvmCompatible = true
 [[platform.java17.dependency]]
 groupId = "io.ballerina.lib"
 artifactId = "avro-native"
-version = "1.0.0"
-path = "../native/build/libs/avro-native-1.0.0.jar"
+version = "1.1.1"
+path = "../native/build/libs/avro-native-1.1.1-SNAPSHOT.jar"
```
Comment (MohamedSabthar) [inline ballerina/Ballerina.toml]: Shouldn't this be avro-native-1.1.0-SNAPSHOT.jar?

**Review by AzeemMuzammil — DISMISSED** [review]
It's version as `1.1.1`, is this intentional?

**Review by AzeemMuzammil — DISMISSED** [review]

File: gradle.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=io.ballerina.lib
-version=1.0.1-SNAPSHOT
```
Comment (ayeshLK) [inline gradle.properties]: Do we we have to bump a minor version for this ?

**Review by TharmiganK — APPROVED** [review]

**Pull request merged**